### PR TITLE
Bump libplanet version to 0.8.0-nightly.20200121

### PR DESCRIPTION
**Pull request by moreal**
In this PR, it applied a lot of features and added `genesis-block` option to apply genesis block feature.

### CODE REVIEW

File: Libplanet.Explorer.Executable/Program.cs [hunk 1]
```diff
@@ -142,7 +146,19 @@ public static async Task Main(string[] args)
             await swarm.PreloadAsync(
                 dialTimeout: TimeSpan.FromSeconds(15),
                 trustedStateValidators: trustedPeers,
-                cancellationToken: cancellationToken
+                cancellationToken: cancellationToken,
+                blockDownloadFailed: (obj, args) =>
+                {
+                    foreach (var exception in args.InnerExceptions)
+                    {
+                        Log.Debug("hello world!");
```
Comment (dahlia) [inline Libplanet.Explorer.Executable/Program.cs]: 👀

File: Libplanet.Explorer.Executable/Program.cs [hunk 2]
```diff
@@ -39,6 +41,8 @@ public static async Task Main(string[] args)
                 .WriteTo.Console();
             Log.Logger = loggerConfig.CreateLogger();
 
+            Log.Debug(options.GenesisBlock.Hash.ToString());
```
Comment (dahlia) [inline Libplanet.Explorer.Executable/Program.cs]: 👀

File: Libplanet.Explorer.Executable/Program.cs (hunk 1 above)
Comment (longfin) [inline Libplanet.Explorer.Executable/Program.cs]: Hello!

File: Libplanet.Explorer.Executable/Program.cs [hunk 3]
```diff
@@ -142,7 +146,19 @@ public static async Task Main(string[] args)
...
+                cancellationToken: cancellationToken,
+                blockDownloadFailed: (obj, args) =>
+                {
+                    foreach (var exception in args.InnerExceptions)
+                    {
+                        Log.Debug("hello world!");
+                        if (exception is InvalidGenesisBlockException invalidGenesisBlockException)
+                        {
+                            Log.Error(invalidGenesisBlockException.Stored.ToString());
+                            Log.Error(invalidGenesisBlockException.NetworkExpected.ToString());
```
Comment (dahlia) [inline Libplanet.Explorer.Executable/Program.cs]: Printing out two lines of block hashes without any explanation?

**Review by dahlia — CHANGES_REQUESTED** [review]

File: Libplanet.Explorer.Executable/Program.cs (hunk 3 above)
Comment (longfin) [inline Libplanet.Explorer.Executable/Program.cs]: Is it okay if we just do logging?

Comment (moreal) [reply Libplanet.Explorer.Executable/Program.cs]: I added the lines to debug exceptions, but we can see an error message in console already. So I removed it again.

Comment (moreal) [reply Libplanet.Explorer.Executable/Program.cs]: I removed the lines again 🔨

File: Libplanet.Explorer.Executable/Program.cs (hunk 2 above)
Comment (dahlia) [reply Libplanet.Explorer.Executable/Program.cs]: Should this be left?

Comment (moreal) [reply Libplanet.Explorer.Executable/Program.cs]: I removed it.

**Comment by moreal** [comment]
The `genesis-block` became to receive absolute path or HTTP URL of the genesis block.

File: Libplanet.Explorer.Executable/Program.cs [hunk 4]
```diff
@@ -1,13 +1,15 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Diagnostics;
```
Comment (dahlia) [inline Libplanet.Explorer.Executable/Program.cs]: What is this for?

**Review by dahlia — APPROVED** [review]

**Review by earlbread — APPROVED** [review]

File: Libplanet.Explorer.Executable/Program.cs (hunk 4 above)
Comment (moreal) [reply Libplanet.Explorer.Executable/Program.cs]: It's unexpected line. I removed it.

**Review by longfin — APPROVED** [review]

**Review by limebell — APPROVED** [review]

**Pull request merged**